components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/admin/SAMLSSOConfigAdmin.java

```diff
@@ -113,7 +136,17 @@ public SAMLSSOServiceProviderDTO addSAMLServiceProvider(SAMLSSOServiceProviderDT
                 String message = "A Service Provider with the name: " + issuer + " is already loaded from the file system.";
                 throw buildClientException(CONFLICTING_SAML_ISSUER, message);
             }
-            return persistSAMLServiceProvider(serviceProviderDO);
+            SAMLSSOServiceProviderDTO samlssoServiceProviderDTO = persistSAMLServiceProvider(serviceProviderDO);
+            if (ApplicationMgtUtil.isLegacyAuditLogsDisabledInAppMgt()){
```

Comment:
```suggestion
            if (ApplicationMgtUtil.isLegacyAuditLogsDisabledInAppMgt()) {
```
formatting issue. check other places as well

Reply:
fixed